composer.json

```diff
@@ -65,7 +70,8 @@
   },
   "config": {
     "allow-plugins": {
-      "magento-hackathon/magento-composer-installer": true
+      "magento-hackathon/magento-composer-installer": true,
+      "cweagans/composer-patches": true
```

Comment:
Seems not required?

Reply:
Don't we need it for recursive patch install if defined by a dependency?
I think it was added automatically by requiring some modules and (since it would keep asking to allow the plugin, which would break the workflow) I left it it.
I'll try to rebuild it to check if it keep being added.

Reply:
you were right, removed